dts/common/yaml/i2c.yaml

```diff
@@ -0,0 +1,25 @@
+---
+title: I2C Base Structure
+id: i2c
+version: 0.1
+
+description: >
+    This binding gives the base structures for all I2C devices
+
+properties:
+  - clock-frequency :
+      type: int
+      category: optional
+      description: Desired I2C bus clock frequency in Hz
```

Comment:
Again here, I used part of the description in linux kernel DT bindings for I2C (https://github.com/torvalds/linux/blob/master/Documentation/devicetree/bindings/i2c/i2c-stm32.txt)